<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#InIPAddress[state=&quot;0&quot;]{
background-color: rgb(255, 0, 0);
}
#InIPAddress[state=&quot;1&quot;]{
	background-color: rgb(85, 255, 0);
}</string>
  </property>
  <property name="state" stdset="0">
   <string>1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Connection Information</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>IP Address</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Port Number</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="InIPAddress">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>30</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
       <property name="inputMask">
        <string>000.000.000.000;_</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
       <property name="state" stdset="0">
        <string/>
       </property>
      </widget>
      <widget class="QSpinBox" name="spnPort">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>70</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1024</number>
       </property>
       <property name="maximum">
        <number>66535</number>
       </property>
      </widget>
      <widget class="QPushButton" name="Button_Connect">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>110</y>
         <width>121</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 255, 255);</string>
       </property>
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
      <widget class="QListWidget" name="lstConsole">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>20</y>
         <width>256</width>
         <height>361</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>400</y>
         <width>261</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="Group_Send_Data">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>160</y>
         <width>301</width>
         <height>191</height>
        </rect>
       </property>
       <property name="title">
        <string>Send Data</string>
       </property>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Message</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="InMessage">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>30</y>
          <width>121</width>
          <height>22</height>
         </rect>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
        <property name="state" stdset="0">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="Button_Send">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>100</y>
          <width>121</width>
          <height>24</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 255, 255);</string>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>lstConsole</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>520</x>
     <y>449</y>
    </hint>
    <hint type="destinationlabel">
     <x>498</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
